.dependency_license

```diff
@@ -0,0 +1,112 @@
+# Documentation - These files are never code.
+README(\..*)?$, Docs
+NOTICE$, Docs
+VERSION$, Docs
+BUILD$, Docs
+DISCLAIMER, Docs
+CHANGELOG$, Docs
+\.md$, Docs
+\.txt$, Docs
+\.placeholder$, Empty
+
+# Uncommentable files
+#
+# These files cannot contain comments, so no header can be added.
+# They are nevertheless covered by the Apache license.
+\.gitignore$, Apache
+\.gitattributes$, Apache
+\.classpath$, Apache
+\.pmd$, Apache
+\.project$, Apache
+^\.rat-excludes$, Apache
+RobotoSlab-Bold\.ttf$, Apache
+RobotoSlab-Regular\.ttf$, Apache
+\.cfg$, Apache
+\.json$, Apache
+\.csv$, Apache
+\.conf$, Apache
+\.prop$, Apache
+\.test$, Apache
+\.config(\.example)?$, Apache
+/\.bowerrc$, Apache
+/\.jshintrc$, Apache
+\.traffic_ops$, Apache # JSON with a specific format.
+\.dist$, Apache # JSON with a specific format.
+traffic_monitor_config\.js$, Apache # Actually JSON with a specific format.
+CrStates$, Apache # JSON with a specific format.
+^docs/.*\.(png|jpe?g|ico|gif)$, Apache
+^traffic_ops/app/public/.*\.(png|jpe?g|ico|gif)$, Apache
```

Comment:
I dont think this is a safe wildcard. Someone could accidentally commit an image into that directory which is not Apache licensed.

Reply:
Hrm, you're probably right. I'll dig into this one.